commcare_connect/opportunity/views.py

```diff
@@ -340,7 +340,7 @@ def get_queryset(self):
 @org_member_required
 def export_user_visits(request, **kwargs):
     opportunity_id = kwargs["pk"]
-    get_object_or_404(Opportunity, organization=request.org, id=opportunity_id)
+    get_opportunity_or_404(org_slug=kwargs["org_slug"], pk=opportunity_id)
```

Comment:
why does `request.org` no longer work? For all the views in this commit, it would be nice to add the `org_slug` arg to the function signature rather than pulling it from `kwargs`, so `def export_user_visits(request, org_slug, **kwargs):` or similar

Reply:
That was my mistake for not realizing earlier that req.org.org_slug is the same as org_slug. I apologize for that. I will make the necessary changes.

Reply:
I won't block on this, because I see that we use this pattern elsewhere, but what I was trying to say was that the function based views would be a lot cleaner if they included the arg names explicitly in the signature, rather than using `kwargs` to get the org slug. Like these views for example https://github.com/dimagi/commcare-connect/blob/main/commcare_connect/opportunity/views.py#L348

There should also be a way to do that with the class based views but I don't have an easy example at hand

Reply:
Ok I will create a new PR for these improvements soon.